configs/components/cfpropertylist.json

```diff
@@ -0,0 +1 @@
+{"url": "git@github.com:ckruse/CFPropertyList.git", "ref": "2.2.7"}
```

Comment:
hate, but if we're not enforcing that for other components (and we're not) then 👍